src/globus_sdk/services/flows/client.py

```diff
@@ -448,6 +448,47 @@ def delete_flow(
 
         return self.delete(f"/flows/{flow_id}", query_params=query_params)
 
+    def get_run(
+        self, run_id: UUIDLike, *, include_flow_description: bool = False
+    ) -> GlobusHTTPResponse:
+        """
+        Retrieve information about a particular Run of a Flow
+
+        :param run_id: The ID of the run to get
+        :type run_id: str or UUID
+        :param include_flow_description: If set to true, the lookup will attempt to
+           attach metadata about the flow to the run to the run response under the key
+           "flow_description"
+        :type include_flow_description: bool, optional - default False
```

Comment:
Sure; happy to fall in line with whatever doc standards make most sense.

Can you give me a suggestion of what you'd like the `type:...` line to look like for an optional param like this?

Reply:
I would keep it to `bool, optional`, and put the default at the end of the `:param:` so that it's included somewhere.

Technically, `optional` isn't part of the type in this case, but it is in cases where we have `str | None` (`str, optional`). We've been documenting any non-required arguments with an `optional` "type" as a kind of white lie.

I don't have a fully thought out game plan for getting these simplified and harmonized across all methods. If you have strong (or weak) opinions about it, let me know and we can discuss it.

Reply:
I don't really have opinions, no.
It's updated to `bool, optional` now